Update USAGE.md

fredericrous opened the pull request:
remove internal address

fredericrous committed:
Update USAGE.md

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SAP/jenkins-pipelayer?pullRequest=2) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/SAP/jenkins-pipelayer?pullRequest=2) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP/jenkins-pipelayer?pullRequest=2) it.</sub>

fredericrous committed:
fix UnsupportedOperationException + documentation

setLibrary.groovy could throw an error at line 25 `libs.add(lib)`,
if there was no library already set (which should not occur)

add documentation to configure the library quickly via jenkins console

Commit:
Merge pull request #1 from fredericrous/master

fix UnsupportedOperationException + documentation
